system-test/spanner.ts

```diff
@@ -1383,15 +1406,12 @@ describe('Spanner', () => {
               Accents ARRAY<STRING(1024)>,
               PhoneNumbers ARRAY<INT64>,
               HasGear BOOL,
-            ) PRIMARY KEY(SingerId)`
+            ) PRIMARY KEY(SingerId)`,
+          GAX_OPTIONS
         )
         .then(onPromiseOperationComplete);
     });
 
-    after(() => {
-      return table.delete().then(onPromiseOperationComplete);
-    });
-
```

Comment:
I'm not sure I understand why this is deleted.

Reply:
The table will be deleted together with the database in after all clean up section.
One less administrative call.